models/demos/metal_BERT_large_11/tt/embeddings.py

```diff
@@ -153,7 +154,7 @@ def __call__(
         token_type_ids.deallocate()
 
         if self.position_embedding_type == "absolute":
-            inputs_plus_token_type_embeddings_tt_tensor = ttl.tensor.add(
+            inputs_plus_token_type_embeddings_tt_tensor = ttnn.add(
```

Comment:
@arakhmati this PR could be way smaller if I was smarter and created an alias. I would use some help here.
W/o an alias, one has to update all tests and all models, which is extremely involved.
Instead, I propose to carry an alias until we are close to the completion of the migration. Wdyt?

Most of the changes one sees in this PR is a switch from `ttl.tensor.add/mul/sub...` or `tt_lib.add..` to ttnn analog.